examples/app.jsx

```diff
@@ -189,6 +190,14 @@ var squareStyles = {
     ':hover': {
       background: 'red',
     }
+  },
+  three: {
+    ':hover': {
+      background: 'yellow'
+    }
+    , ':disabled': {
```

Comment:
Yeah, I don't think we lint the examples. We could add it one of these days, I suppose.